### PR TITLE
Node SDK Example

### PR DESCRIPTION
**Pull request by jmsktm**

### CODE REVIEW

File: node-sdk-demo/package.json
Comment (jmsktm) [inline node-sdk-demo/package.json:30]: Didn't need the sdk because of local linking. But needs to be added before merge (while testing)

File: node-sdk-demo/test/workspaceApi.test.js
Comment (foxish) [inline node-sdk-demo/test/workspaceApi.test.js:19]: don't need this?

File: node-sdk-demo/test/workspaceApi.test.js
Comment (foxish) [inline node-sdk-demo/test/workspaceApi.test.js:23]: I'm surprised that this is no different than the JSON based payload directly sent to the API - is there no intermediate objects etc?

File: node-sdk-demo/test/workspaceApi.test.js
Comment (foxish) [inline node-sdk-demo/test/workspaceApi.test.js:95]: we should find a way to avoid putting the key everywhere

**Comment by foxish** [comment]
Oops, sorry - was deleting my own comment and deleted yours by mistake.

File: node-sdk-demo/test/workspaceApi.test.js
Comment (foxish) [reply node-sdk-demo/test/workspaceApi.test.js:23]: I can see some of these objects do exist - for example: https://github.com/signadot/node-sdk/blob/main/src/model/WorkspaceFork.js#L26

File: node-sdk-demo/test/workspaceApi.test.js
Comment (foxish) [inline node-sdk-demo/test/workspaceApi.test.js:70]: do these console.log statements need to be removed?

Comment (jmsktm) [reply node-sdk-demo/test/workspaceApi.test.js:70]: It was helpful at times during development. So we can choose to keep it or remove it. Do you suggest removing them?

**Review by foxish — APPROVED** [review]
LGTM, thanks! Ok to merge after testing once end to end.

**Comment by jmsktm** [comment]
Tested prior to merge.

**Pull request merged**
